frontend/src/views/WheresMyGene/common/store/reducer.ts

```diff
@@ -164,17 +164,14 @@ function selectOrganism(
     return state;
   }
 
+  const { sortBy, snapshotId, xAxisHeight } = state;
+
   return {
-    ...state,
-    selectedGenes: [],
+    ...INITIAL_STATE,
```

Comment:
I think @signechambers1 mentioned in #5777 that we want to reset all custom control stuff, which I assume includes sortBy! (It's a duplicate ticket if #5778)

I was able to remove xAxisHeight from the new state, since the new height needs to be recalculated anyway, so no point passing it here?

Reply:
@seve just wanna make sure you saw this! Thanks!

Reply:
saw it!